pandas/io/pytables.py

```diff
@@ -2297,9 +2298,9 @@ class Fixed(StringMixin):
         parent : my parent HDFStore
         group  : the group node where the table resides
         """
-    pandas_kind = None
-    obj_type = None
-    ndim = None
+    pandas_kind = None  # type: Optional[str]
+    obj_type = None  # type: Optional[Type[Union[DataFrame, Series]]]
+    ndim = None  # type: Optional[int]
```

Comment:
Similar question on usage after declaration; this is required to be an int right?

Reply:
Yes. `pandas_kind` and `obj_type` don't appear to actually be optional either, so I'll fix that as well.